core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -98,11 +99,11 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
   errorMessage!: string;
   explorationContentHtml!: string | string[];
   finalCommitMessage!: string;
-  isCurriculumAdmin: boolean = false;
-  isHtmlContent: boolean = false;
+  curriculumIsAdmin: boolean = false;
```

Comment:
This should be userIsCurriculumAdmin.

Did you not read [this comment](https://github.com/oppia/oppia/pull/15089#discussion_r835872374)? I already mentioned the correct replacement names there.

Reply:
Ohh, sorry for that, working on 3 PR simultaneously, so became confused.
Done now